src/monitoring/bin/db_full_check.sh

```diff
@@ -1,8 +1,18 @@
+#!/bin/bash
+
+set -euo pipefail
+
 echo CROMWELL_GOOGLE_PROJECT is ${CROMWELL_GOOGLE_PROJECT}
 echo CROMWELL_CLOUDSQL_INSTANCE is ${CROMWELL_CLOUDSQL_INSTANCE}
 echo CROMWELL_MAXIMUM_DATABASE_SIZE_TIB is ${CROMWELL_MAXIMUM_DATABASE_SIZE_TIB}
 echo CROMWELL_DATABASE_FORECAST_LIMIT_MONTHS is ${CROMWELL_DATABASE_FORECAST_LIMIT_MONTHS}
-# echo CROMWELL_METRICS_ACCESS_TOKEN is ${CROMWELL_METRICS_ACCESS_TOKEN}
+if [[ ! -z ${CROMWELL_METRICS_ACCESS_TOKEN+x} ]]; then
+  echo CROMWELL_METRICS_ACCESS_TOKEN has been defined
```

Comment:
TOL: You maybe don't need this "you did it right!" case, just the error if it's not there?